ui/frontend/types.ts

```diff
@@ -36,7 +36,8 @@ export interface CommonEditorProps {
 
 export enum Editor {
   Simple = 'simple',
-  Advanced = 'advanced',
```

Comment:
Doing it this way will require changes to local_storage.ts to preserve people's current selections. If you don't feel like making those changes, you could also leave the value as `advanced` and solely add monaco.

Reply:
I added a migration to local_storage.ts.